docs/flask.md

```diff
@@ -0,0 +1,118 @@
+# Flask-GraphQL
+
+Adds GraphQL support to your Flask application.
+
+## Installation
+
+To install the integration with Flask, run the below command on your terminal.
+
+`pip install graphql-server-core[flask]`
+
+## Usage
+
+Use the `GraphQLView` view from `graphql_server.flask`.
+
+```python
+from flask import Flask
+from graphql_server.flask import GraphQLView
+
+from schema import schema
+
+app = Flask(__name__)
+
+app.add_url_rule('/graphql', view_func=GraphQLView.as_view(
+    'graphql',
+    schema=schema,
+    graphiql=True,
+))
+
+# Optional, for adding batch query support (used in Apollo-Client)
+app.add_url_rule('/graphql/batch', view_func=GraphQLView.as_view(
+    'graphql',
+    schema=schema,
+    batch=True
+))
+
+if __name__ == '__main__':
+    app.run()
+```
+
+This will add `/graphql` and `/graphiql` endpoints to your app.
+
+### Special Note for GraphQL-Server V3
+
+If you are using the `Schema` type of [Graphene](https://github.com/graphql-python/graphene) library, be sure to use the `graphql_schema` attribute to pass as schema on the `GraphQLView` view. Otherwise, the `GraphQLSchema` from `graphql-core` is the way to go.
+
+**Graphene Schema**
+
+```python
+from graphene import ObjectType, String, Schema
+
+
+class Query(ObjectType):
+    # this defines a Field `hello` in our Schema with a single Argument `name`
+    hello = String(name=String(default_value="stranger"))
+    goodbye = String()
+
+    # our Resolver method takes the GraphQL context (root, info) as well as
+    # Argument (name) for the Field and returns data for the query Response
+    @staticmethod
+    def resolve_hello(root, info, name):
+        return f'Hello {name}!'
+
+    @staticmethod
+    def resolve_goodbye(root, info):
+        return 'See ya!'
+
+# Use the `graphql_schema` if using `Schema` type
+schema = Schema(query=Query).graphql_schema
+```
+
+**Graphql-Core Schema**
+
+```python
+from graphql import GraphQLSchema, GraphQLObjectType, GraphQLField, GraphQLString
+
+
+schema = GraphQLSchema(
+    query=GraphQLObjectType(
+        name='RootQueryType',
+        fields={
+            'hello': GraphQLField(
+                GraphQLString,
+                resolve=lambda obj, info: 'world')
+        }))
+```
```

Comment:
I’m not sure how relevant it is to have these examples on how to define a graphene and graphql-core schema. It might be better to just link out to the relevant documentation on it.

Reply:
Was thinking on the same thing but now that you created a wiki for Graphene v3 release, I can replace this code block for something like [this](https://github.com/graphql-python/graphene/wiki/v3-release-notes#graphene-schema-no-longer-subclasses-graphqlschema-type).